Apps/Laboratory/src/Delegates/RestLaboratoryDelegate.cs

```diff
@@ -312,5 +312,104 @@ public async Task<RequestResult<PHSAResult<IEnumerable<CovidTestResult>>>> GetPu
             this.logger.LogDebug($"Finished getting public COVID-19 test results {phn} {dateOfBirth} {collectionDate}...");
             return retVal;
         }
+
+        /// <inheritdoc/>
+        public async Task<RequestResult<RapidTestResponse>> SubmitRapidTestAsync(string hdid, string bearerToken, AuthenticatedRapidTestRequest rapidTestRequest)
+        {
+            using (Source.StartActivity("SubmitRapidTestAsync"))
```

Comment:
Nit: our preferred style for `using` statements is the C# 8.0 syntax that removes the need for braces:

`using Source.StartActivity("SubmitRapidTestAsync");`